pynamodb/connection/base.py

```diff
@@ -178,7 +179,8 @@ class Connection(object):
     A higher level abstraction over botocore
     """
 
-    def __init__(self, region=None, host=None, session_cls=None):
+    def __init__(self, region=None, host=None, session_cls=None,
+                 request_timeout_seconds=None, max_retry_attempts=None, base_backoff_ms=None):
```

Comment:
let's rename to max_attempts since i think this variable encompasses the initial attempt as well